api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -86,7 +86,6 @@ message VmConfig {
   // (proxy_on_start). `google.protobuf.Struct` is serialized as JSON before
   // passing it to the plugin. `google.protobuf.BytesValue` and
   // `google.protobuf.StringValue` are passed directly without the wrapper.
-  // [#extension-category: envoy.wasm.runtime]
```

Comment:
not sure about the current placement - but iiuc then this needs to be placed somewhere

its not just for docs - its also going to be used for generating jsonschema from the protos - so its necessary for traversing to the correct configuration etc

Reply:
all the thing from extension-category attribute is already written manually https://github.com/envoyproxy/envoy/blob/938c52daa380b78e226548187844027ea313fe99/api/envoy/extensions/wasm/v3/wasm.proto#L53-L79 with extra information. make sense?

Reply:
> its also going to be used for generating jsonschema from the protos - so its necessary for traversing to the correct configuration etc

I see..

Reply:
but it looks a bit ugly if I place it `runtime` field: https://github.com/envoyproxy/envoy/blob/938c52daa380b78e226548187844027ea313fe99/api/envoy/extensions/wasm/v3/wasm.proto#L53-L79 since the info duplicates between them (manually written one vs auto generated "tips" section). wdyt?

Reply:
i think remove the manual one in that case - this is a necessary part of the proto architecture

Reply:
not sure what to do about the additional information - i guess either add beneath or move to the linked protos

Reply:
> but it looks a bit ugly 

regarding this - the aesthetics are not great currently

once i get round to it im intending to improve the ui/x here - probably putting links, examples, deprecation info, metadata etc into tabs

Reply:
ok thanks! will fix accordingly tomorrow